server/asset/dcr/dcr_test.go

```diff
@@ -765,7 +765,7 @@ func TestUTXOs(t *testing.T) {
 	// Set the value of this one.
 	txout.Value = 2.0
 	// There is no block info to add, since this is a mempool transaction
-	utxo, err := dcr.utxo(txHash, msg.vout, nil)
+	utxo, err := dcr.utxo(context.Background(), txHash, msg.vout, nil)
```

Comment:
I think [in other tests](https://github.com/decred/dcrdex/blob/fb6f3eaa7dae7d043aae8ceb5771638554dd5d1c/client/asset/dcr/dcr_test.go#L494-L503), we've created a `context.WithCancel` in `TestMain` for these purposes.

Reply:
`testBackend()` pretty  much does that for the tests individually.  But since this isn't a "live" test, these context go nowhere anyway.